<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QcParamSettingsWidget</class>
 <widget class="QWidget" name="QcParamSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1659</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QcParamSettingsWidget</string>
  </property>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>338</y>
     <width>1151</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>1155</x>
     <y>338</y>
     <width>1</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1556</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>339</y>
     <width>1</width>
     <height>449</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>787</y>
     <width>1151</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>338</y>
     <width>384</width>
     <height>450</height>
    </rect>
   </property>
   <property name="borders" stdset="0">
    <string>all</string>
   </property>
   <widget class="QCheckBox" name="checkBoxQcTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>质控时间</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxQcIntervSamp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>间隔样本数</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxQcIntervHours">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>质控间隔时间</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAfterCali">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>校准后</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>339</y>
     <width>1151</width>
     <height>449</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>284</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>45</number>
   </attribute>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1539</width>
     <height>159</height>
    </rect>
   </property>
   <property name="title">
    <string>联合质控</string>
   </property>
   <property name="borders" stdset="0">
    <string>group_tit</string>
   </property>
   <widget class="QWidget" name="widget10" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>78</y>
      <width>771</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>批时间限</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditQcbatchLimit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>小时</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="line_h_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>792</y>
     <width>1556</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1</width>
     <height>793</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>168</y>
     <width>1539</width>
     <height>166</height>
    </rect>
   </property>
   <property name="title">
    <string>自动质控</string>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>29</y>
      <width>1491</width>
      <height>135</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelQcTime">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>质控时间</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEditQcTime">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>72</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>40</y>
       <width>190</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>间隔样本数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditQcIntervSamp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>72</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>质控间隔时间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditQcIntervHours">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>72</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>72</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>小时</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>578</x>
       <y>72</y>
       <width>911</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_week1">
        <property name="text">
         <string>星期一</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_week2">
        <property name="text">
         <string>星期二</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_week3">
        <property name="text">
         <string>星期三</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_week4">
        <property name="text">
         <string>星期四</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_week5">
        <property name="text">
         <string>星期五</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_week6">
        <property name="text">
         <string>星期六</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_week0">
        <property name="text">
         <string>星期天</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>tableView</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>line_h_2</zorder>
  <zorder>line_h</zorder>
  <zorder>line_v</zorder>
  <zorder>line_v_3</zorder>
  <zorder>line_h_3</zorder>
  <zorder>line_h_4</zorder>
  <zorder>line_v_4</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditQcbatchLimit</tabstop>
  <tabstop>lineEditQcIntervSamp</tabstop>
  <tabstop>lineEditQcIntervHours</tabstop>
  <tabstop>timeEditQcTime</tabstop>
  <tabstop>checkBox_week1</tabstop>
  <tabstop>checkBox_week2</tabstop>
  <tabstop>checkBox_week3</tabstop>
  <tabstop>checkBox_week4</tabstop>
  <tabstop>checkBox_week5</tabstop>
  <tabstop>checkBox_week6</tabstop>
  <tabstop>checkBox_week0</tabstop>
  <tabstop>checkBoxQcTime</tabstop>
  <tabstop>checkBoxQcIntervSamp</tabstop>
  <tabstop>checkBoxQcIntervHours</tabstop>
  <tabstop>checkBoxAfterCali</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
